src/libraries/System.Private.CoreLib/src/System/IO/Stream.cs

```diff
@@ -212,7 +212,9 @@ public virtual ValueTask DisposeAsync()
             }
             else
             {
+#pragma warning disable CA1416 // Validate platform compatibility, need proper handling
```

Comment:
Please create an issue and link it, we have to block the API or fix it

Reply:
Created an issue https://github.com/dotnet/runtime/issues/44543